ui.apps/package.json

```diff
@@ -8,9 +8,9 @@
     "test": "npm run prettier:check && npm run karma:run",
     "karma:run": "karma start --single-run --browsers ChromeHeadless,FirefoxHeadless karma.conf.js",
     "karma:debug": "karma start --single-run=false --browsers Chrome --debug karma.conf.js",
-    "prettier:check": "prettier --check 'src/**/*.{js,css}' --config ./.prettierrc",
-    "prettier:fix": "prettier --write 'src/**/*.{js,css}' --config ./.prettierrc",
-    "webpack:build": "webpack --config webpack.config.js --mode development"
+    "prettier:check": "prettier --check '{src,test}/**/*.{js,css}' --config ./.prettierrc",
```

Comment:
Is there a reason to have a check and a fix? Can't we just fix the formatting right away?

Reply:
CircleCi should fail when executing the check, but not do the fix itself, because this would require that it commits the formatted code. So the workflow is:
* Write code
* Check locally with `prettier:fix`
* Commit and push
* CircleCi runs `prettier:check` and fails if not properly formatted